lib/studies/reply/search_page.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+
+class SearchPage extends StatelessWidget {
+  const SearchPage();
+
+  @override
+  Widget build(BuildContext context) {
+    final splashRadius = 24.0;
+
+    return Scaffold(
+      body: SafeArea(
+        child: Material(
+          color: Theme.of(context).colorScheme.surface,
+          child: Column(
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Row(
+                  mainAxisSize: MainAxisSize.max,
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    IconButton(
+                      icon: const Icon(Icons.arrow_back),
+                      onPressed: () => Navigator.of(context).pop(),
+                      splashRadius: splashRadius,
```

Comment:
Why does the splashRadius need to be overridden?

Reply:
It doesn't but the Android splashRadius looked like it was much smaller. I think to keep everything consistent I won't override `splashRadius`.